pipeline/objects/function.py

```diff
@@ -53,11 +53,16 @@ def __init__(
 
     @classmethod
     def from_schema(cls, schema: FunctionGet):
-        unpickled_data = hex_to_python_object(schema.hex_file.data)
+        unpickled_data = load_object(schema.hex_file.data)
```

Comment:
I presume we can't get away from the `hex_file` name due to backwards-compatability?

Reply:
Indeed 😞